src/scripts/pokeballs/Pokeballs.ts

```diff
@@ -98,6 +98,12 @@ class Pokeballs implements Feature {
                 return Math.min(15,Math.pow(amountCaught,2) / 5000);
             }, 1250, 'Increased catch rate with more catches', new RouteKillRequirement(10, GameConstants.Region.johto, 34)),
 
+            new Pokeball(GameConstants.Pokeball.Beastball, () => {
+                if (typeof GameConstants.UltraBeastType[Battle.enemyPokemon().name] === 'number') {
+                    return 10;
+                }
```

Comment:
Don't think we need this code anymore, as ultra balls can only be used on Ultra beasts. It's confusing to have code that don't do anything